python/README.rst

```diff
@@ -0,0 +1,18 @@
+=====
+pyddq
+=====
+
+Python binding to Drunken Data Quality
+
+
+Description
+===========
+
+A LONGER DESCRIPTION OF YOUR PROJECT GOES HERE...
```

Comment:
Should we provide something here, @Gerrrr ? Or is it not used? I can copy something from the general readme.